addons/purchase_requisition/wizard/purchase_requisition_create_alternative.py

```diff
@@ -60,6 +60,8 @@ def action_create_alternative(self):
             )
         vals = self._get_alternative_values()
         alt_po = self.env['purchase.order'].with_context(origin_po_id=self.origin_po_id.id, default_requisition_id=False).create(vals)
+        for line in alt_po.order_line:
+            line._compute_tax_id()
```

Comment:
a detail but the method has a loop itself so I think we could do

```suggestion
        alt_po.order_line._compute_tax_id()
```

Reply:
Good catch, this has been changed accordingly !